wagtail/admin/checks.py

```diff
@@ -178,7 +244,7 @@ def wagtail_admin_base_url_check(app_configs, **kwargs):
                 "The WAGTAILADMIN_BASE_URL setting is not defined",
                 hint="This should be the base URL used to access the Wagtail admin site. "
                 "Without this, URLs in notification emails will not display correctly.",
-                id="wagtailadmin.W003",
+                id="wagtailadmin.W005",
```

Comment:
I don't think this error code should have changed

Reply:
@NXPY123 are you able to look at this question.

Reply:
Sorry about that it's a mistake on my part. I've changed it back now.